configs/nncf_compression/vehicle-detection/vehicle-detection-0200.py

```diff
@@ -0,0 +1,159 @@
+# model settings
```

Comment:
should not it be in training_extensions?

Reply:
Yes, sir!

Reply:
Fixed